src/main/java/com/teamc/fms/service/FacilityService.java

```diff
@@ -0,0 +1,20 @@
+package com.teamc.fms.service;
+
+import com.teamc.fms.entity.Facility;
+import com.teamc.fms.repo.FacilityRepository;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+
+public interface FacilityService {
+
+    public abstract List<Facility> getFacilities();
+
+    public abstract void addFacility(Facility facility);
+
+    public abstract void deleteFacility(String id);
+
+    public abstract void updateFacility(Facility facility);
```

Comment:
It is redundant to add public abstract in interfaces, as the default methods in interface is public abstract
Also please use Java DOCs to document the methods, even the logic is simple, to be consisitent